### DISCUSSION TITLE
6.01萬魔殿任務中文化

### DISCUSSION THREAD

**Discussion by Manbbb** [GpointChen/FFXIVChnTextPatch-GP#146] [Q&A] [answered]
請問使用0416發佈的更新包
6.01萬魔殿的任務跟NPC還沒有中文是正常的嗎？
感謝大大，辛苦了

![ScreenHunter 816](https://user-images.githubusercontent.com/103889052/163717538-d44ce1a4-c8e2-4b5c-9cce-f0bb82051b80.jpg)

**Comment by GpointChen** [comment, marked as answer]
正常，中國服進度只到6.00。

**Reply by Manbbb** [reply]
明白，好的^^因為看WIKI時看到日期以為已經發佈了
那再等一下好了，感謝大大你的更新及整理，Thank you.

![image](https://user-images.githubusercontent.com/103889052/163851171-789f5ee7-ce02-413b-984f-f8cdc7281e92.png)

**Reply by GpointChen** [reply]
確實是發佈了，只是我最近才更新。感謝你的提醒。
